src/backend/looped/vehicle/seatbelt.cpp

```diff
@@ -10,13 +10,13 @@ namespace big
 		virtual void on_tick() override
 		{
 			PED::SET_PED_CONFIG_FLAG(self::ped, 32, false);
-			PED::SET_PED_CAN_BE_KNOCKED_OFF_VEHICLE(self::ped, false);
+			PED::SET_PED_CAN_BE_KNOCKED_OFF_VEHICLE(self::ped, 1);
 		}
 
 		virtual void on_disable() override
 		{
 			PED::SET_PED_CONFIG_FLAG(self::ped, 32, true);
-			PED::SET_PED_CAN_BE_KNOCKED_OFF_VEHICLE(self::ped, true);
+			PED::SET_PED_CAN_BE_KNOCKED_OFF_VEHICLE(self::ped, 0);
```

Comment:
They are boolean values, use `true/false` instead of `1/0`.

Reply:
> They are boolean values, use `true/false` instead of `1/0`.

`SET_​PED_​CAN_​BE_​KNOCKED_​OFF_​VEHICLE(Ped ped, int state)`
It's not a boolean, it has 4 different states:

```
enum eKnockOffVehicle
{
	KNOCKOFFVEHICLE_DEFAULT,
	KNOCKOFFVEHICLE_NEVER,
	KNOCKOFFVEHICLE_EASY,
	KNOCKOFFVEHICLE_HARD
};
```

Reply:
Then define the enum and use it? instead of hardcoding a 0